content/docs/start/experiments.md

```diff
@@ -211,12 +211,257 @@ scores.json  auc       0.61314  0.07139
 And finally, we can compare `ROC` curves with a single command!
 
 ```dvc
+$ dvc plots modify prc.json -x recall -y precision
+$ git commit -am "Update plots format"
+
 $ dvc plots diff
 file:///Users/dvc/example-get-started/plots.html
 ```
```

Comment:
What's the point/need for this addition? This is no longer "a single command".

Would also need to update the image underneath. I think we want to keep this simple, no need for modify here.

Reply:
@jorgeorpinel it's needed indeed to get the image below :) May be we can change or move it somehow (e.g. hide into the expandable section).

Reply:
Adding a full `dvc plots modify` to the guide just so the plot axis labels match the image presented doesn't seem super necessary to me. But OK, up to you. Will just need to change the "with a single command" text above, and fix the plots modify expandable section below (per https://github.com/iterative/dvc.org/pull/1514#pullrequestreview-441093000).

Reply:
Oh I see it's not just the labels but the actual fields used as data sources. OK

Reply:
Well, we could just add -x and -y to the plots diff command, actually. See https://dvc.org/doc/command-reference/plots/diff#options